src/test-clang.bats

```diff
@@ -254,6 +254,14 @@ testBuildHello() {
   expectedSuffix=
 }
 
+@test "loong64-c-ld" {
+  skip "LOONG64 not supported"
```

Comment:
Why is this not working? Doesn't debian provide installable packages? How are users supposed to get it working?

Reply:
After completing, the modification is completed, and the test can currently be tested